packages/popup-component/src/components/popup/popup.tsx

```diff
@@ -58,6 +69,14 @@ export class DcPopup {
             <span aria-hidden="true">&times;</span>
           </button>
         </div>
+        <button
+          type="button"
+          class="notification__action"
+          aria-label="got it"
```

Comment:
Great to see you using a11y patterns! I would suggest instead “got it” something like “popup confirmation” just to give more context by itself

Reply:
Great, thanks dude! That's much better